build.sbt

```diff
@@ -12,16 +12,6 @@ ThisBuild / scalacOptions ++= Seq(
 ThisBuild / libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.9" % Test
 ThisBuild / libraryDependencies += "org.scalacheck" %% "scalacheck" % "1.15.4" % Test
 ThisBuild / libraryDependencies += "org.scala-lang.modules" %% "scala-parser-combinators" % "2.2.0"
-lazy val picolib = RootProject(
-  uri("https://github.com/hmc-cs111-spring2023/picolib.git#no-graphics")
-)
-
-lazy val root = project
-  .settings(
-    name := "rollMaker" // you can change this to be the name of your language
-  )
-  .in(file("."))
-  .dependsOn(picolib)
 
```

Comment:
Removing dependency on this library. I don't think you'll need it for your DSL, and it makes things simpler to remove it.